src/main/java/org/folio/dao/util/ConnectionProvider.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.dao.util;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import org.folio.rest.persist.PostgresClient;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import static java.lang.String.format;
+
+/**
+ * Util class to obtain single connection to underlying database using DriverManager.
+ * Connection pool is not provided.
+ */
+public class ConnectionProvider {
+  private static final String JDBC_DRIVER = "jdbc:postgresql";
+  private static final String CONFIG_USERNAME_KEY = "username";
+  private static final String CONFIG_PASS_KEY = "password";
+  private static final String CONFIG_HOST_KEY = "host";
+  private static final String CONFIG_PORT_KEY = "port";
+  private static final String CONFIG_DATABASE_KEY = "database";
+
+  private ConnectionProvider() {
+  }
+
+  /**
+   * Returns database connection for the given tenant
+   *
+   * @param vertx  vertx instance
+   * @param tenant given tenant to which schema connection should be provided
+   * @return Connection to database
+   * @throws SQLException if error occurs while trying to obtain access to database
+   */
+  public static Connection getConnection(Vertx vertx, String tenant) throws SQLException {
```

Comment:
we are not using a connection pool. That will lead to a lot an expensive operation of fetching connection each time